css/selectors/descendant.json

```diff
@@ -9,12 +9,28 @@
             "webview_android": {
               "version_added": true
             },
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "61",
+                "notes": [
```

Comment:
There should be `"partial_implementation": true` between this and the previous line, eg.:
```json:
{
  "version_added": "61",
  "partial_implementation": true,
  "notes": [
```